.github/workflows/beta-release.yml

```
@@ -0,0 +1,100 @@
+          mkdir -p android/app/keystore
+          echo ${{ secrets.KEYSTORE_BASE64 }} | base64 -d > android/app/keystore/mybudget_key.jks
+
+      - name: Flutter pub get
+        run: flutter pub get
+
+      - name: Build apk
+        run: flutter build apk --flavor beta --release --no-tree-shake-icons
+
+      - name: Extract version from pubspec.yaml
+        id: get_version
+        run: |
```

actionlint (1.7.8)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)